src/main/scala/org/dbpedia/lookup/server/Result.scala

```diff
@@ -33,7 +33,7 @@ class Result(uri: String,
 
     // make a category label from a category URI
     def categoryLabel(categoryUri: String): String = {
-        WikiUtil.wikiDecode(categoryUri.replace("http://dbpedia.org/resource/Category:", ""))
+        WikiUtil.wikiDecode(categoryUri.replace("http://fr.dbpedia.org/resource/Catégorie:", ""))

```

Comment:
Not sure if hard-coding prefixes is the way to go? Would it not be
better to get prefixes like this from a config parameter?

Am 04.02.2016 um 09:13 schrieb Sandro:
>
> In src/main/scala/org/dbpedia/lookup/server/Result.scala
> <https://github.com/dbpedia/lookup/pull/3#discussion_r51840523>:
>
> > @@ -33,7 +33,7 @@ class Result(uri: String,
> >  
> >      // make a category label from a category URI
> >      def categoryLabel(categoryUri: String): String = {
> > -        WikiUtil.wikiDecode(categoryUri.replace("http://dbpedia.org/resource/Category:", ""))
> > +        WikiUtil.wikiDecode(categoryUri.replace("http://fr.dbpedia.org/resource/Catégorie:", ""))
>
> Hardcoded URI // We would like to receive it from a properties file
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dbpedia/lookup/pull/3/files#r51840523>.
>